packages/ui/vue-ui/src/main.ts

```diff
@@ -8,4 +11,5 @@ export default {
     },
 };
 
-export { CrossmintPaymentElement };
+export { CrossmintPaymentElement, useCrossmintEvents, CheckoutEvents };
```

Comment:
we shouldnt export stuff thats already exported from other packages?

Reply:
hmmm why not? I want to make it easy to the user to get all these packages from vue sdk, and don't have to also import the sdk base package.

Similarly when you work with react, that you import everything from one place